cosmwasm-plugin/main.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"embed"
+	"encoding/gob"
+	"fmt"
+	"io"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	hplugin "github.com/hashicorp/go-plugin"
+
+	"github.com/ignite/cli/ignite/services/chain"
+	"github.com/ignite/cli/ignite/services/plugin"
+)
+
+var (
+	chainName  string
+	chainNamed string
+	//go:embed wasm-wiring/*
+	templates embed.FS // Embedded template files
+
+)
+
+func init() {
+	gob.Register(plugin.Manifest{})
+	gob.Register(plugin.ExecutedCommand{})
+	gob.Register(plugin.ExecutedHook{})
+}
+
+type p struct{}
+
+func (p) Manifest() (plugin.Manifest, error) {
+	return plugin.Manifest{
+		Name: "cosmwasm-plugin",
```

Comment:
I think it would be safe to rename `cosmwasm-plugin` to `cosmwasm` so it would look like `ignite cosmwasm add`.

The plugin folder name can also be renamed to `cosmwasm`.

Also it would be nice to have at least a short description. I think you can ignore the long one if you want.

Reply:
Renamed plugin to cosmwasm: https://github.com/ignite/plugins/commit/c46f39cf2008c4e1588b61dd19884504779a381a